### ISSUE TITLE
Questionable obsolete message

### ISSUE THREAD

**Issue by SimonCropp** [NLog/NLog#745]
So I have `logger.Debug( message, exception);`

And I get 

```
Error: 'NLog.Logger.Debug(string, System.Exception)' is obsolete: 'Use Debug(LogLevel level, Exception exception, string message, params object[] args) method instead.'	
```

But seems i should be using `logger.Debug(exception, message);` and not `Debug(LogLevel level, Exception exception, string message, params object[] args)`.

So am I mistaken or is the obsolete wrong?

**Comment by 304NotModified** [comment]
Thanks. This is indeed a bug.

**Comment by SimonCropp** [comment]
Want a PR?

**Comment by 304NotModified** [comment]
That would be great!

**Comment by 304NotModified** [comment]
Note: The bug could be in the `.tt` files

**Comment by SimonCropp** [comment]
done

**Comment by swalter** [comment]
I think the same currently happens do "Fatal"

**Comment by 304NotModified** [comment]
Also fatal was fixed in #751. Or is there another issue?

**Comment by swalter** [comment]
Just saw that you only switched the parameters => fixed.
